tests/conftest.py

```diff
@@ -1373,26 +1373,28 @@ def skip_test_if_no_ocs_sc(ocs_storage_class):
 
 
 @pytest.fixture(scope="session")
-def skip_test_if_no_nfs_sc(cluster_storage_classes_names):
+def verify_nfs_sc(cluster_storage_classes_names):
```

Comment:
here and in other places I would suggest to rename `skip_test_if` to `fail_if`
```suggestion
def fail_if_no_nfs_sc(cluster_storage_classes_names):
```

Reply:
+1